src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/HealthCertificateOverviewViewController.swift

```diff
@@ -93,7 +93,11 @@ class HealthCertificateOverviewViewController: UITableViewController {
 		navigationController?.navigationBar.prefersLargeTitles = true
 		navigationController?.navigationBar.sizeToFit()
 
-		tableView.reloadData()
+		// delay reloadData because scrolling to top may take up to 0.35 sec
```

Comment:
Just out of curiosity what scrolling to top? ^^

Reply:
it that special logic, what triggers an embedded scrollview to scroll on top if we tap an tabBarItem again